contracts/Validators.sol

```diff
@@ -1188,4 +1193,44 @@ contract Validators is
     function getPoolManager(address val) external view returns (address) {
         return poolInfos[val].manager;
     }
+
+
+    function recoverGoDaofunds() external onlyAdmin {
+        
+        // GoDao funds stuck in kcc staking contract
+        // 
+        // mainnet: 
+        //   https://scan.kcc.io/tx/0xf55066470ac141f7ff053e6c37d278b8019db1e7a5ea5a23da462c4b509830be 
+        //   - 1,100,000 KCS sent to 0x000000000000000000000000000000000000F333 
+        //   - only 110,000 KCS are used for the Ishikari Hardforks 
+        //   - The remainning 1,100,000 - 110,000 = 990,000 KCS should be returned to GoDao
+        //
+        // testnet: 
+        //  https://scan-testnet.kcc.network/tx/0x5c0d15fae86b4010adffae8e8732b4229240afb3e8dd7cc87c799ff2fe2dd0c9
+        //   - 1 KCS directly sent to 0x000000000000000000000000000000000000F333
+        //   - This "1 KCS" should be returned to GoDao 
+        //
```

Comment:
As there is no funds stuck in the testnet contract, I directly sent 1KCS for later testing in testnet.